### PR TITLE
ipyopt

### PR DESCRIPTION
**Pull request by minrk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by minrk** [comment]
I believe this is ready to go @conda-forge/help-python-c

File: recipes/ipyopt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config
+  host:
+    - python >=3
```
Comment (marcelotrevisani) [inline recipes/ipyopt/meta.yaml]: Do you want to skip python 3? If you, I think it is better to skip it in the build section, as you are using the selectors to skip for windows

File: recipes/ipyopt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5a105ef68227304c0bb81c7583cf9ea83d27bfc3584a7a2e531fdef991d49e03
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (marcelotrevisani) [inline recipes/ipyopt/meta.yaml]:
```suggestion
  skip: true  # [win or py2k]
```

Comment (minrk) [reply recipes/ipyopt/meta.yaml]: Aha! I had `or py2` and it wasn't working. Will fix.

File: recipes/ipyopt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    - pkg-config
+  host:
+    - python
+    - ipopt
+    - numpy 1.14
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('ipopt') }}
+    - numpy
```
Comment (marcelotrevisani) [inline recipes/ipyopt/meta.yaml]: are you compiling agains ``numpy``? if so, please change it to:
```suggestion
    - {{ pin_compatible('numpy') }}
```

Comment (minrk) [reply recipes/ipyopt/meta.yaml]: 👍 I expected numpy to have run_exports!

**Pull request merged**